doc/src/docs/InputOutputReference/01d-InputOutputReference.md

```diff
@@ -19370,6 +19370,75 @@ An example IDF showing how this object is used is provided below:
 
 There are no output variables reported for the DemandManager:Thermostats object.
 
+
+### DemandManager:Ventilation
+
+The DemandManager:Ventilation object is used for limiting the ventilation rate calculated from the Controller:OutdoorAir object.
+
+#### Field: Name
+
+The name of the DemandManager:Ventilation object.
+
+#### Field: Availability Schedule Name
+
+The reference to the schedule object specifying the availability of this demand manager. A schedule value of zero indicates that this demand response (DR) is not applicable for that time period. A schedule greater than zero indicates that the demand response applies during the time period. If this field is blank, the schedule has values of one for all time period.
+
+#### Field: Limit Control
+
+This field specifies the type of limiting control. There are three options. The **FixedRate** option reduces the ventilation to a specified air flow rate. The **ReductionRatio** specifies the multiplier of the ventilation rate. The **Off** option disables the demand manager in the simulation.
+
+#### Field: Minimum Limit Duration
+
+The minimum amount of time [minutes] that the demand manager will continue to demand limit after being activated. This prevent loads from turning on and off every time step.
+
+#### Field: Fixed Rate
+
+This field specifies the amount of fixed ventilation rate when the demand manager is active and *FixedRate* limit control is applied. The unit is m3/s.
+
+#### Field: Reduction Ratio
+
+This field specifies the multiplier of the ventilation rate when the demand manager is active and *ReductionRatio* limit control is applied.
+
+#### Field: Limit Step Change
+
+NOT YET IMPLEMENTED.
```

Comment:
So is there a plan for this ```Limit Step Change``` field to be implemented?  If not then I'd like to remove it.  These stubs aren't necessary.  When that capability is added later, we can add it to the end of the object or a simple transition rule can insert it.  @mjwitte?